src/services/codefixes/fixAddMissingEnumMember.ts

```diff
@@ -0,0 +1,79 @@
+/* @internal */
+namespace ts.codefix {

```

Comment:
i would merge this with `fixAddMissingMember.ts` do not see why we need to separate code fixes here.